Description: Add /usr/share/snmp/mibs to MIB search path
Forwarded: not-needed
Origin: vendor, http://anonsvn.wireshark.org/viewvc?view=rev&revision=31052
Author: Balint Reczey <balint@balintreczey.hu>

Index: b/epan/oids.c
===================================================================
--- a/epan/oids.c	2010-02-15 17:23:16.000000000 +0100
+++ b/epan/oids.c	2010-02-15 17:24:58.000000000 +0100
@@ -1077,12 +1077,10 @@
 #else
 #define PATH_SEPARATOR ":"
 	path = smiGetPath();
-#ifdef __APPLE__
 	g_string_append(path_str, "/usr/share/snmp/mibs");
 	if (strlen(path) > 0 ) {
 		g_string_append(path_str, PATH_SEPARATOR);
 	}
-#endif
 	g_string_append_printf(path_str, "%s", path);
 	free (path);
 #endif
